spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'food index page', type: :feature do
+
+  it 'displays the total number of search results for a given term' do
+    visit '/'
+
+    fill_in "q", with: "sweet potatoes"
+
+    click_on "Search"
+
+    expect(current_path).to eq("/foods")
+
+    expect(page).to have_content("Total Results: 49676")
```

Comment:
For an assessment this is totally fine. If this were a project I would warn against using this exact value in the assertion unless you're stubbing the API call with Webmock and using a static fixture. Otherwise, if the API changes, this number could change and break your test